src/components/Entry.js

```diff
@@ -3,27 +3,29 @@ import styled from "styled-components";
 
 const StyledEntry = styled.div`
   display: flex;
-  margin: auto;
-  padding: 4px;
+  padding: 10px;
   background: #ebebeb;
   border-radius: 4px;
-  width: 300px;
+  margin: 10px;
+} 
 
   span {
-    flex: 1;
     font-size: 15px;
     color: #636363;
-    margin: auto;
-    padding: 4px;
-    text-align: center
+    text-align: center;
+  }
+  
+  .entry-amount {
+    margin-left: 15px;
+    font-weight: 600;
   }
 `;
 
 export default function Entry(props) {
   return (
     <StyledEntry>
       <span>{props.title}</span>
-      <span>${props.amount}</span>
+      <span className= 'entry-amount'>${props.amount}</span>
```

Comment:
Strange that `prettier` didnt clean up the space between `className=` and ` 'entry'`. try running `npm install` before the next time you do `npm start` to make sure prettier is installed